opentelemetry-sdk/tests/metrics/test_metrics.py

```diff
@@ -194,17 +194,17 @@ def test_register_asynchronous_instrument(
 
         meter_provider._measurement_consumer.register_asynchronous_instrument.assert_called_with(
             meter_provider.get_meter("name").create_observable_counter(
-                "name", Mock()
+                "name0", Mock()
             )
         )
         meter_provider._measurement_consumer.register_asynchronous_instrument.assert_called_with(
             meter_provider.get_meter("name").create_observable_up_down_counter(
-                "name", Mock()
+                "name1", Mock()
             )
         )
         meter_provider._measurement_consumer.register_asynchronous_instrument.assert_called_with(
             meter_provider.get_meter("name").create_observable_gauge(
-                "name", Mock()
+                "name2", Mock()
```

Comment:
Does it make sense to test for duplicate instruments in the SDK tests as well to ensure the exception is raised? This could cover the case of potential regressions where someone forgets to call the parent's method.

Reply:
Sure, can't hurt to have these tests, added. :+1: